### DISCUSSION TITLE
Is there a command for listing the most recently modified entries?

### DISCUSSION THREAD

**Discussion by Serenade600** [jrnl-org/jrnl#1881] [Q&A]
I know the command to edit the most recent entries is like `jrnl -n 10 --edit`. But could I edit the most recently MODIFIED entries by typing a command maybe looking like `jrnl -n 10 -modified --edit`?

**Comment by micahellison** [comment]
No, jrnl doesn't track the modification date of entries, so there's no way to do this in the client.

Theoretically, something like this could be done in the directory format, but it would still be based on the file modification time for a whole day, so it wouldn't be very reliable whenever there are multiple entries in a day.
